.github/workflows/workflow.yaml

```diff
@@ -6,11 +6,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-10.15]
+        os: [ubuntu-latest, macos-11]
```

Comment:
This update seems to trigger CI to fail with an error that looks similar to https://github.com/tweag/rules_haskell/issues/1607. It's not clear to me at the moment if this is really an issue within rules_nixpkgs or an issue in nixpkgs upstream. Either way, I think this PR shouldn't be blocked on it. It's better to get these examples merged sooner and record the issue to be fixed separately.

Could you switch this one back to the working MacOS version, if the example can't be made to work with MacOS 10.15 then just disable the MacOS test for now, and then open an issue to keep track of this problem.

IIRC the MacOS update was needed because of a more recent nixpkgs needed in the examples because of some compatibility issue. Could you also document this chain of dependencies?